src/main/java/com/minecolonies/coremod/client/gui/WindowTownHall.java

```diff
@@ -631,9 +637,11 @@ private void createAndSetStatistics()
             return;
         }
 
+        final int maxCitizens =  ColonyManager.getColonyByPosFromDim(townHall.getColony().getDimension(),townHall.getColony().getCenter()).getCitizenManager().getMaxCitizens();
```

Comment:
This isnt going to work in multiplayer, the GUI's are clientside and you can only access the ColonyView there, not the actual colony object.

Reply:
Yes, @boraozarslan you gotta sync it with the colonyView

Reply:
Ah, I see... I'll look into it